src/miner_hbbft_sidecar.erl

```diff
@@ -181,10 +181,10 @@ handle_call({submit, Txn}, From,
                                     case blockchain_txn:absorb(Txn, Chain) of
                                         ok ->
                                             spawn(fun() ->
-                                                        %% this will now return {ok, Position, Length}
-                                                        %% or {error, full} which we could feed back to the caller using gen_server:reply() on the From
-                                                        {ok, QueuePos, QueueLen} = libp2p_group_relcast:handle_command(Group, {txn, Txn}),
-                                                        gen_server:reply(From, {ok, QueuePos, QueueLen, Height})
+                                                        %% this will now return {ok, Position, Length} or {error, full}
+                                                        %% which we feed back to the caller using gen_server:reply() on the From
+                                                        Reply = libp2p_group_relcast:handle_command(Group, {txn, Txn}),
+                                                        gen_server:reply(From, {Reply, Height})
```

Comment:
I note you have changed the tuple format of the response here with the height outside of the {ok, ...} tuple.  No objections to that but i believe there are a couple of other replys that will need updated to maintain consistency